lib/deserialize.js

```diff
@@ -22,28 +22,56 @@ module.exports = function (app) {
     });
 
     if (matches.length > 0) {
-
-      //set the JSON API Content-Type response header
+      // set the JSON API Content-Type response header
       ctx.res.set({'Content-Type': 'application/vnd.api+json'});
```

Comment:
A side note, we should be able to remove these Content-Type header sets soon as LB should now support switching on the Accept header. 

I plan on expanding loopbacks native support for handling the json api content type soon as well